packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -53,6 +60,12 @@ export const COMPOSITE_ONLY_ICONS = {
   /* @conditional-compile-remove-from(stable) Chat_Notification_Icon*/
   ControlBarButtonBadgeIcon: <Circle20Filled />,
   /* @conditional-compile-remove-from(stable): meeting/calling-composite */
+  ControlBarChatButtonFilled: <Chat20Filled />,
```

Comment:
don't use `filled` and `normal` here, use naming that derives contextual meaning, e.g. for screenshare we use ControlButtonScreenShare*Start/Stop*.
Consider ControlBarChatButtonSelected/Unselected ControlBarChatButtonOpen/Closed.

The reason being is contoso has no idea when filled or normal is used (or really what filled and normal are - their icons may not be "filled"